schema/run.host.event.yaml

```diff
@@ -0,0 +1,58 @@
+---
+$id: runhost
+$schema: http://json-schema.org/draft-07/schema#
+type: object
+properties:
+  event_type:
+    type: string
+    enum:
+    - create
+    - read
+    - update
+    - delete
+  hostPayload:
+    type: object
+    properties:
+      id:
+        type: string
+      run_id:
+        type: string
+      inventory_id:
+        type: string
+      host:
+        type: string
+      log:
+        type: string
+      events:
+        type: string
+      sat_sequence:
+        type: int32
```

Comment:
events and sat_sequence are internal fields which should not be exposed via this interface (analogously we don't expose these via REST's /run_hosts resource either)